scripts/hudson/narayana.sh

```diff
@@ -411,20 +411,27 @@ function clone_as {
 }
 
 function build_as {
-  echo "Building JBoss EAP/WildFly"
+  REPO="https://github.com/wildfly/wildfly.git"
+  AS_BRANCH=27.0.0.Alpha1
```

Comment:
I will import this tag into the Narayana fork

Reply:
It's there now: https://github.com/jbosstm/jboss-as/tree/27.0.0.Alpha1

Reply:
Something to consider is that 27.0.0.Alpha1 is no longer supported by WildFly I expect

Reply:
Hopefully we won't need this tag for long if we quickly move to support native jakarta.
My primary goal is being able to create narayana releases.
I am cloning WildFly directly and ignoring 5_BRANCH (but like I say hopefully we won't need my workarounds for long).

Reply:
I think any changes related to cloning the AS should go into the clone_as function: https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh#L355

Reply:
I put some comments about the complexity/performance trade-off and why simplification should trump generality. But if it is going to help getting the PR passed then I can re-instate some of the complexity.

Reply:
If clone_as function remains then I think adding cloning functionality to the build_as function will add complexity.